newsfragments/4182.doc.rst

```diff
@@ -0,0 +1 @@
+Changed ``versionadded`` for "Type information included by default" feature from ``v68.3.0`` to ``v69.0.0`` -- by :user:Avasam`
```

Comment:
@Avasam looks like you forgot a backtick here and now rendering is broken in the changelog. You can fix this by making a PR updating the changelog directly.

Reply:
Ironic, my PR to fix the changelog will need a PR to fix the changelog ^^"